recipes/libpqxx/all/conanfile.py

```diff
@@ -30,7 +30,12 @@ class LibpqxxConan(ConanFile):
 
     @property
     def _min_cppstd(self):
-        return 14 if Version(self.version) < "7.0" else "17"
+        if Version(self.version) < "7.0":
+            return 14
+        elif Version(self.version) < "7.8.0":
+            return 17
+        else:
+            return 20
```

Comment:
I've tried to locally reproduce your issue but have not been able to do so i Conan v2.
This might be an issue with Conan 1, as this code does not run there, you would need to change the function just underneath this one to control which compilers get used in v1.

Nevertheless, https://github.com/jtv/libpqxx/releases in 7.8.0 and 7.8.1 the changelog seems to imply that C++ 17 should be enough to compile, so I wonder what the actual build issue is here 🤔

Reply:
I encountered the same problem as in https://github.com/jtv/libpqxx/issues/743 .  The reason is that Conan v1, unlike v2 version, it doesn't specify ```compiler.cppstd``` in the profile by default, so if libpqxx is compiled with C++ 17 and your project uses C++ 20 or newer it will fail to build, but if libpqxx is compiled with C++ 20 and your project uses C++ 20 or newer it will be compiled successfully